rqpy/plotting/_iv_didv_tools_plotting.py

```diff
@@ -103,121 +139,146 @@ def _plot_rload_rn_qetbias(IVanalysisOBJ, lgcsave, xlims_rl, ylims_rl, xlims_rn,
     if ylims_rn is not None:
         axes[1].set_ylim(ylis_rn)
 
-    axes[0].errorbar(IVanalysisOBJ.vb[0,0,IVanalysisOBJ.scinds]*1e6,
-                     np.array(IVanalysisOBJ.rload_list)*1e3, 
-                     yerr = IVanalysisOBJ.rshunt_err*1e3, linestyle = '', marker = '.', ms = 10)
+    axes[0].errorbar(
+        IVanalysisOBJ.vb[0,0,IVanalysisOBJ.scinds]*1e6,
+        np.array(IVanalysisOBJ.rload_list)*1e3,
+        yerr=IVanalysisOBJ.rshunt_err*1e3,
+        linestyle='',
+        marker='.',
+        ms=10,
+    )
     axes[0].grid(True, linestyle = 'dashed')
     axes[0].set_title('Rload vs Vbias', fontsize = 14)
     axes[0].set_ylabel(r'$R_ℓ$ [mΩ]', fontsize = 14)
     axes[0].set_xlabel(r'$V_{bias}$ [μV]', fontsize = 14)
-    axes[0].tick_params(axis="both", direction="in", top=True, right=True, which="both")
-
-    axes[1].errorbar(IVanalysisOBJ.vb[0,0,IVanalysisOBJ.norminds]*1e6,
-                     np.array(IVanalysisOBJ.rtot_list)*1e3, 
-                     yerr = IVanalysisOBJ.rshunt_err*1e3, linestyle = '', marker = '.', ms = 10)
-    axes[1].grid(True, linestyle = 'dashed')
-    axes[1].set_title('Rtotal vs Vbias', fontsize = 14)
-    axes[1].set_ylabel(r'$R_{N} + R_ℓ$ [mΩ]', fontsize = 14)
-    axes[1].set_xlabel(r'$V_{bias}$ [μV]', fontsize = 14)
-    axes[1].tick_params(axis="both", direction="in", top=True, right=True, which="both")
+    axes[0].tick_params(
+        axis="both", direction="in", top=True, right=True, which="both",
+    )
+
+    axes[1].errorbar(
+        IVanalysisOBJ.vb[0,0,IVanalysisOBJ.norminds]*1e6,
+        np.array(IVanalysisOBJ.rtot_list)*1e3,
+        yerr=IVanalysisOBJ.rshunt_err*1e3,
+        linestyle='',
+        marker='.',
+        ms=10,
+    )
+    axes[1].grid(True, linestyle='dashed')
+    axes[1].set_title('Rtotal vs Vbias', fontsize=14)
+    axes[1].set_ylabel(r'$R_{N} + R_ℓ$ [mΩ]', fontsize=14)
```

Comment:
Does this labeling work with the ℓ and Ω characters? Or should we switch to their LaTeX versions (\ell, \Omega)

```suggestion
    axes[1].set_ylabel(r'$R_{N} + R_\ell$ [m$\Omega$]', fontsize=14)
```

Reply:
I know that Python 3 should support it, but sometimes the LaTeX compiler errors on the special characters in between $'s

Reply:
changed this, and a few more places as well